src/libmongoc/tests/json-test.c

```diff
@@ -1121,7 +1122,10 @@ check_outcome_collection (mongoc_collection_t *collection, bson_t *test)
       mongoc_collection_set_read_concern (collection, rc);
    }
 
-   cursor = mongoc_collection_find_with_opts (collection, &query, NULL, prefs);
+   bson_init (&opts);
+   BCON_APPEND (&opts, "sort", "{", "_id", BCON_INT32 (1), "}");
+   cursor = mongoc_collection_find_with_opts (collection, &query, &opts, prefs);
+   bson_destroy (&opts);
```

Comment:
To confirm, this will be split out into its own commit before merging? I noticed CDRIVER-3666 still wasn't listed in https://github.com/mongodb/mongo-c-driver/pull/648/commits (and the JIRA issue is just "open" with no link to this PR) so wanted to double check. OK if so, in which case you can probably move the JIRA ticket right to closed once its commit is merged.

Reply:
Yes, I will revise the commits before merging. I was worried that force pushing to this PR could mess up the current conversations. Marked CDRIVER-3666 as in code review and commented with a link to this PR.